### ISSUE TITLE
3.2.6: Using "should" in the Understanding doc is confusing

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1874]
The Understanding doc says that the linked help "should" be accessible and gives a non-exhaustive list as to how this may be done.

We are concerned that this may be interpreted as too loose a suggestion. Whilst we understand from https://github.com/w3c/wcag/issues/1237#issuecomment-664958802 that it's not possible in 2.x to "scope-in" the external help, the Understanding doc is non-normative, so it would be possible to use stronger language in the document to alert readers to how important it is for the help content to be accessible.

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by mraccess77** [comment]
My personal comment is that this SC does not require help - just that when help is required it is in a relative order.  Given that the help already may or may not be present on the page and is not required by the SC it would be out of scope to go too deep in describing how to provide accessible help.  Perhaps an external resource article could be linked if one exists.

**Comment by patrickhlauke** [comment]
indeed, while it may seem weak to use "should" here, changing this to a "must" will give the mistaken impression that this SC also requires/mandates other things more directly about how help must be implemented. if the help itself is included in an assessment/in scope of an accessibility statement, then it will naturally be subject to all other SCs as well.

at a higher level, i actually don't think it's helpful that the understanding document focuses so much on explaining what the various help types could be - it's nothing to do with the actual SC itself, and it just leaves a weird imbalance in the understanding document as it doesn't really focus on what auditors should do to pass/fail this SC, but gives lots of weak "should" suggestions on how to provide help on a site, which is NOT the point of the SC. i filed https://github.com/w3c/wcag/issues/1881 separately just now for this.

**Comment by alastc** [comment]
Keeping to the original topic, chair-hat-on: There is no way we can use 'must' for something that is not required by the SC, that will generate objections from other W3C members (this is a known issue that has come up before). In fact, if possible we should avoid using "should", so rather than:

> The location in a smaller viewport may be different than in a larger viewport but the mechanism or link should be consistent across a set of web pages. The location should remain consistent both visually and programmatically. 

Something like:
> The location in a smaller viewport may be different than in a larger viewport but it is best if the mechanism or link is consistent across a set of web pages. A consistent location, both visually and programmatically, is the most usable.

**Comment by DavidMacDonald** [comment]
Suggested response:

The working group cannot use 'must' for something that is not required by the SC. That will generate objections from other W3C members (this is a known issue that has come up before). In fact, if possible we should avoid using "should", so rather than:

> The location in a smaller viewport may be different than in a larger viewport but the mechanism or link should be consistent across a set of web pages. The location should remain consistent both visually and programmatically.

We will change the understanding to:

> The location in a smaller viewport may be different than in a larger viewport but it is best if the mechanism or link is consistent across a set of web pages. A consistent location, both visually and programmatically, is the most usable.
